genesyscloud/orgauthorization_pairing/resource_genesyscloud_orgauthorization_pairing_test.go

```diff
@@ -171,7 +171,7 @@ func TestAccResourceOrgAuthorizationPairing(t *testing.T) {
 						"genesyscloud_group."+groupResource2, "id"),
 					resource.TestCheckResourceAttr("genesyscloud_orgauthorization_pairing."+orgAuthorizationPairingResource, "group_ids.#", "2"),
 					func(s *terraform.State) error {
-						time.Sleep(30 * time.Second) // Wait for 30 seconds for resources to get deleted properly
+						time.Sleep(45 * time.Second) // Wait for 30 seconds for resources to get deleted properly
```

Comment:
Maybe just update the comment so people know it's supposed to be a 45 second wait